qemu_v8.mk

```diff
@@ -65,7 +65,9 @@ arm-tf-clean:
 # QEMU
 ################################################################################
 qemu:
-	cd $(QEMU_PATH); ./configure --target-list=aarch64-softmmu\
+	cd $(QEMU_PATH); \
+		git submodule update --init dtc && \
```

Comment:
@igoropaniuk yeah ignore my first proposal, it's broken because `repo sync` errors out due to `qemu/dtc` exists already.
FYI, I tried what you are asking with my second proposal (https://github.com/OP-TEE/build/pull/156#discussion_r125310828) and it's alright:
```
$ cd eqmu
$ git submodule update --init dtc
Submodule 'dtc' (git://git.qemu-project.org/dtc.git) registered for path 'dtc'
Submodule path 'dtc': checked out '558cd81bdd432769b59bff01240c44f82cfb1a9d'
```